runtime/runtime/src/actions.rs

```diff
@@ -773,13 +809,67 @@ pub(crate) fn check_account_existence(
     Ok(())
 }
 
+#[cfg(feature = "protocol_feature_add_account_versions")]
+lazy_static_include::lazy_static_include_str! {
+    STORAGE_USAGE_DELTA_CSV => "res/storage_usage_delta.csv"
+}
+
+#[cfg(feature = "protocol_feature_add_account_versions")]
+fn read_storage_usage_delta() -> HashMap<AccountId, u64> {
+    let mut reader = csv::Reader::from_reader(STORAGE_USAGE_DELTA_CSV.as_bytes());
+    let mut result = HashMap::new();
+    for record in reader.records() {
+        let record = record.expect("Malformed storage_usage_delta.csv");
+        let account_id = &record[0];
+        let delta = record[1].parse::<u64>().expect("Malformed storage_usage_delta.csv");
+        result.insert(account_id.to_string() as AccountId, delta);
+    }
+    result
+}
+
+#[cfg(feature = "protocol_feature_add_account_versions")]
+lazy_static::lazy_static! {
+    static ref STORAGE_USAGE_DELTA_FROM_FILE: HashMap<AccountId, u64> = read_storage_usage_delta();
```

Comment:
We already did enough hacks around by pulling a random CSV file right into the binary, let's at least have the HashMap properly propagated through the configuration. `neard` should be the single source of configuration, we should not hijack runtime and get changed its operation due to some random global variable. Let `neard` hold the file, parse it conditionally when the chain id is `mainnet`, and pass it through the runtime configs down here if necessary.